src/components/SearchBox.vue

```diff
@@ -81,9 +88,20 @@
               viewBox="0 0 38 38"
               :class="suit('loadingIcon')"
             >
-              <g fill="none" fill-rule="evenodd">
-                <g transform="translate(1 1)" stroke-width="2">
-                  <circle stroke-opacity=".5" cx="18" cy="18" r="18"/>
+              <g
+                fill="none"
+                fill-rule="evenodd"
+              >
+                <g
+                  transform="translate(1 1)"
+                  stroke-width="2"
+                >
+                  <circle
```

Comment:
kind of sad that this doesn't work like prettier but oh well

Reply:
is there an option for minimal # of props before it's moved vertically?

Reply:
Yes it's [1](https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/max-attributes-per-line.md) by default.

Reply:
probably too tricky (with some longer attributes), but I would prefer `singleline: 2` I think

Reply:
This PR doesn't update the general configuration, we can create a PR on [eslint-config-algolia](https://github.com/algolia/eslint-config-algolia) for this.